metadata-db-app/src/cmr/metadata_db/config.clj

```diff
@@ -31,6 +34,33 @@
    (metadata-db-username)
    (metadata-db-password)))
 
+(defn pg-db-spec-primary
+  [connection-pool-name]
+  (aurora/db-spec
+   connection-pool-name
+   (aurora-config/db-url-primary)
+   (metadata-db-username)
+   (metadata-db-password)
+   (aurora-config/aurora-db-name)))
+
+(defn pg-db-spec-secondary
+  [connection-pool-name]
+  (aurora/db-spec
+   connection-pool-name
+   (aurora-config/db-url-secondary)
+   (metadata-db-username)
+   (metadata-db-password)
+   (aurora-config/aurora-db-name)))
+
+;; this db connection method for prototype use only
+(def pooled-pg-db-primary (delay (aurora/make-prototype-pool (pg-db-spec-primary "Metadata DB Primary"))))
```

Comment:
What's the purpose of the delay here?

Reply:
Short explanation -- that's how the docs for our jdbc lib say to do it. (as well as other libs using db connection pools in clojure)
Long explanation -- delay is essentially a wrapper to make something lazy. it appears the reason this is recommended for database connection pools in particular, is because you don't want the pool to initiate until it's called.